Team3749-2021/src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -17,6 +17,7 @@
  * @author Rohan Juneja
  */
 
+// Constructor for Shooter
```

Comment:
I think you put this line in the wrong place (this is the class definition not constructor)